server/datastore/mysql/migrations/tables/20161118212649_CreateTableUsers.go

```diff
@@ -18,7 +18,7 @@ func Up_20161118212649(tx *sql.Tx) error {
 			"`deleted` tinyint(1) NOT NULL DEFAULT FALSE," +
 			"`username` varchar(255) NOT NULL," +
 			"`password` varbinary(255) NOT NULL," +
-			"`salt` varchar(255) NOT NULL," +
+			"`salt` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL," +
```

Comment:
Why is this here in addition to the table default? Seems like we only need one or the other.